src/os/ui/draw/basedrawcontrols.js

```diff
@@ -365,6 +365,11 @@ export class Controller {
    * @export
    */
   activateControl(type) {
+    // Cancel any active measure interaction
+    if (this.measureInteraction && this.measureInteraction.getEnabled()) {
```

Comment:
`this.measureInteraction` isn't defined in the base class. The compiler should have caught this, but we should extend `activateControl` in `drawcontrols.js` for this behavior.